developing/cicd/delivery-pipeline.md

```diff
@@ -70,16 +68,16 @@ it. This workflow is orchestrated via an automated tool like CircleCI, and
 looks something like this:
 
 1. New code is merged into the default branch.
-2. CircleCI detects that the default branch has been updated and deploys the code to
+1. CircleCI detects that the default branch has been updated and deploys the code to
    our `development` environment.
-3. CircleCI checks to make sure the deploy was successful. Is our
+1. CircleCI checks to make sure the deploy was successful. Is our
    environment running the right version of the code?
-4. CircleCI (or another tool fired by CircleCI) performs post-deployment
+1. CircleCI (or another tool fired by CircleCI) performs post-deployment
    testing. Does it pass some functional tests to ensure user workflows
    are functional (eg, can someone log in, pull up a user record, put
    things in a shopping cart, etc)? Are logs filling up with error now
    that the new version is deployed?
-5. If the tests pass, CircleCI goes ahead and deploys to our `production`
+1. If the tests pass, CircleCI goes ahead and deploys to our `production`
```

Comment:
Why is mdformat flagging these ordered lists? Having them all be `1` is not readable. I've noticed multiple instances of this change. Have the changes that mdformat made been reviewed?

Reply:
It sounds like they're doing it to make diffs easier to read: https://mdformat.readthedocs.io/en/stable/users/style.html#ordered-lists

I don't agree with that reasoning. It's the end result that matters since we're producing material for humans to read. We should optimize for readability of the final product, not of the Git diffs. I wonder what others think. Could we solicit feedback before making big changes like this in the future?

It looks like mdformat has a config to allow consecutive numbering. I would vote to use it.

Reply:
I think mdformat assumes you are reading the document using a markdown renderer, as is the case here (since it's hosted on github). This way is also arguably easier to write. So I get where they're coming from.

But, I'm not committed to this at all. I've played around with the consecutive numbering option and it seems like it works pretty well. It even fixes numbering errors, or at least tries.

Reply:
Ah! My bad! I was looking at the rich diff inside this PR, and GitHub rendered the ordered list with the number 1 everywhere. so I assumed that was the actual output. I should have looked at the page directly from the repo, and there I see it ordered with the right numbers.